Dockerfile

```diff
@@ -61,7 +61,7 @@ RUN php -r "copy('https://getcomposer.org/installer', '/tmp/composer-setup.php')
 WORKDIR /var/www/html/
 USER www-data
 
-RUN wget https://github.com/cachethq/Cachet/archive/${cachet_ver}.tar.gz && \
+RUN wget https://codeload.github.com/CachetHQ/Cachet/tar.gz/v${cachet_ver} -O ${cachet_ver}.tar.gz && \
```

Comment:
The idea here is you specify a full Cachet tag, _including_ the `v`, so this should not really be needed.

This seems to have introduced a 404 when testing: https://travis-ci.org/CachetHQ/Docker/builds/221871287#L1316